### PR TITLE
report image is displayed

### PR DESCRIPTION
**Pull request by Lucrezia-Cester**
This PR links to this issue https://github.com/GSTT-CSC/hazen-web-app/issues/58

The image obtained from the hazen task is now displayed in the report page

### CODE REVIEW

**Comment by Lucrezia-Cester** [comment]
I still need to figure out how to migrate the database for this change, as it returns errors.

**Comment by Lucrezia-Cester** [comment]
@heyhaleema , could you please review this PR? [](url)

https://user-images.githubusercontent.com/83493021/229488190-b73c1a80-41bb-4fda-8b4f-f24b566c89ed.mov

It shows how when a task is run the report also displays the image

**Comment by Lucrezia-Cester** [comment]
@tomaroberts, @heyhaleema, could this PR be reviewed please?

File: hazen_reports/Ghosting/Ghosting_ghosting_NSA1_30_1.png
Comment (heyhaleema) [inline hazen_reports/Ghosting/Ghosting_ghosting_NSA1_30_1.png:1]: @Lucrezia-Cester I suggest we add the `hazen_reports/` directory to the `.gitignore`.

File: docker/nginx/hazen.conf [hunk 1]
```diff
@@ -10,6 +10,10 @@ server {
      # To disable buffering
      proxy_buffering off;
 
+     location /static/ {
+        alias /APP/hazen_reports;
```
Comment (heyhaleema) [inline docker/nginx/hazen.conf:14]: @Lucrezia-Cester I was thinking maybe we should create a `data/` directory and write the `uploads` and `hazen_reports/` directories to that so it's easier to `.gitignore` - what do you think?

File: app/static/Ghosting_ghosting_SAG_1nsa_30_1.png
Comment (heyhaleema) [inline app/static/Ghosting_ghosting_SAG_1nsa_30_1.png:1]: @Lucrezia-Cester I think we should leave the `static/` directory for images like the ones we use for the tasks on the front page (vs. result images).

**Review by heyhaleema — COMMENTED** [review]
@Lucrezia-Cester I've had a quick look at this PR and whilst it works when I go the Reports page, I receive an error when trying to see an image's individual report:

https://user-images.githubusercontent.com/67117138/232837775-190d0b5e-b108-43ca-ab71-165acadbd5f0.mov

Also, more aesthetics, what do you think about removing the paths in the Measurement Results and Image Result columns in the Reports page? I feel like it's not necessary for the Image Result, whereas the text in the Measurement Results column could be better formatted. Not necessarily something to be included in this PR as I think this should be it's own issue, but I thought it'd be good to get thoughts in this thread first.

**Pull request closed without merging**